dist/payform.js

```diff
@@ -144,7 +144,7 @@
         luhn: true
       }, {
         type: 'dinersclub',
-        pattern: /^(36|38|30[0-5])/,
+        pattern: /^(36|38|30[0-5])|309/,
```

Comment:
Consider this shorter equivalent:

```js
pattern: /^(36|38|30[0-59])/
```

Reply:
Hi @glebm !
Changed. 
Thanks,
Francesco